website/docs/maintenance/observability/monitor-metrics.md

```diff
@@ -327,7 +327,7 @@ Some metrics might not be exposed when using other JVM implementations (e.g. IBM
     </tr>
     <tr>
       <td>eventQueueSize</td>
-      <td>The number of events waiting to be processed in the queue.</td>
+      <td>The number of events currently waiting to be processed in the coordinator event queue. This metric is labeled with <code>event_type</code> to distinguish between different types of coordinator events.</td>
```

Comment:
We should also update the `eventQueueSize`. Currently there are 2 metrics for `eventQueueSize`, one is directly under `coordinator` which reports the total queue size without `event_type` label. The other reports under the `event` infix with the `event_type` label.